CgfConverter/Renderers/Gltf/Models/GltfBufferView.cs

```diff
@@ -13,5 +13,8 @@ public class GltfBufferView
 
     [JsonProperty("byteOffset")] public long ByteOffset;
 
+    [JsonProperty("target", NullValueHandling = NullValueHandling.Ignore)]
+    public GltfBufferViewTarget? Target;
```

Comment:
Should these be proper properties with get/set (or maybe a record with get/init)?

Reply:
I think enum really is the best fit for this case, as it's exported as an integer in JSON form, and then it is mandatorily optional. The value should be set for vertex and index buffers, but not for everything else, including texture data.